TestsPython/tests/base_test.py

```diff
@@ -1,19 +1,16 @@
 import os
 import sys
 from alttester import AltDriver
-import unittest
 
 sys.path.append(os.path.dirname(__file__))
 
 
-class TestBase(unittest.TestCase):
+class TestBase():
```

Comment:
Here you want to do something similar with what we do in the SDK tests: https://github.com/alttester/AltTester-Unity-SDK/blob/development/Bindings~/python/tests/integration/conftest.py#L50 and https://github.com/alttester/AltTester-Unity-SDK/blob/development/Bindings~/python/tests/integration/test_scene01.py#L29